frontend/src/components/nodes/NodeNav.tsx

```diff
@@ -26,7 +26,7 @@ export default class extends React.Component<Props> {
                 >
                   <Col className="node-selector align-self-center">
                     {context.nodeInfo.validatorAmount
-                      ? `${context.nodeInfo.validatorAmount}  Validating`
+                      ? `${context.nodeInfo.validatorAmount}  Validating & New Upcoming`
                       : `- Validating`}
```

Comment:
Here you got bitten by the code duplication. Your messages are now different during the loading and the loaded states.